content/en/docs/ops/setup/standalone-operator/index.md

```diff
@@ -1,196 +1,134 @@
+`IstioControlPlane` resource and then install the Istio components corresponding
+to the specified (`demo`) configuration.
 
-{{< tab name="default" cookie-value="default" >}}
-This profile enables Istio's default settings which contains recommended
-production settings. Run the following command to switch to this profile:
+You can confirm the Istio control plane services have been deployed with the following commands:
 
 {{< text bash >}}
-$ kubectl apply -f https://preliminary.istio.io/operator-profile-default.yaml
+$ kubectl get svc -n istio-system
+NAME                     TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                                                                                                                                      AGE
+grafana                  ClusterIP      172.21.211.123   <none>          3000/TCP                                                                                                                                     2m
+istio-citadel            ClusterIP      172.21.177.222   <none>          8060/TCP,15014/TCP                                                                                                                           2m
+istio-egressgateway      ClusterIP      172.21.113.24    <none>          80/TCP,443/TCP,15443/TCP                                                                                                                     2m
+istio-galley             ClusterIP      172.21.132.247   <none>          443/TCP,15014/TCP,9901/TCP                                                                                                                   2m
+istio-ingressgateway     LoadBalancer   172.21.144.254   52.116.22.242   15020:31831/TCP,80:31380/TCP,443:31390/TCP,31400:31400/TCP,15029:30318/TCP,15030:32645/TCP,15031:31933/TCP,15032:31188/TCP,15443:30838/TCP   2m
+istio-pilot              ClusterIP      172.21.105.205   <none>          15010/TCP,15011/TCP,8080/TCP,15014/TCP                                                                                                       2m
+istio-policy             ClusterIP      172.21.14.236    <none>          9091/TCP,15004/TCP,15014/TCP                                                                                                                 2m
+istio-sidecar-injector   ClusterIP      172.21.155.47    <none>          443/TCP,15014/TCP                                                                                                                            2m
+istio-telemetry          ClusterIP      172.21.196.79    <none>          9091/TCP,15004/TCP,15014/TCP,42422/TCP                                                                                                       2m
+jaeger-agent             ClusterIP      None             <none>          5775/UDP,6831/UDP,6832/UDP                                                                                                                   2m
```

Comment:
Is this copied from somewhere or actual output from trying it recently? The reason I ask is we don't have jaeger agent at the moment in istioctl, so this install seems a bit different.

Reply:
@sdake added this, but I think he just copied it from the quick start instructions. If it's wrong, I think it needs to be updated in both places.

Reply:
The charts are still somewhat in flux - we should have a definitive output tomorrow sometime. Let's hold off merging this till then and update the other docs at the same time.

Reply:
ya - this was followup work that needed to be done.  :)